recipes/blosc-hdf5-plugin/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "blosc-hdf5-plugin" %}
+{% set version = "1.0" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+  path: src/ 
```

Comment:
This is not a great way to distribute code because the bot won't be able to update it.

Reply:
I do not believe, that there will be any updates in the future, since this interface is very stable. The plugin basically is just a binding, the last PR there has been merged like three years ago. If you insist, I can create a separate repository (fork).

Reply:
Where does this plugin come from, then?

Reply:
Yeah, If you need make significant changes, it might not be fair to call it the same thing.  Have you tried getting those changes upstream?

I would recommend *just* putting the cmake files in the recipe and then in the build script copying them over, overwriting the parts of code downloaded from https://github.com/Blosc/hdf5-blosc

Reply:
How would I specify this exactly? When I use a git_url, non of the recipe files will be deployed to the build directory. Right now I changed it to include the orignal repo, and patching their CMakeLists via an ugly patch.

Reply:
It is a little annoying because they don't have tags, but it looks like v1.0.1 was released as commit 574c7d22265434b90978c0e7aeb5829ad0b19490, so you can use the URL https://github.com/Blosc/hdf5-blosc/archive/574c7d22265434b90978c0e7aeb5829ad0b19490.tar.gz to get a tarball